### PR TITLE
How to set up studies - add ANY relevant info

### PR DESCRIPTION
**Pull request by sineknorr**

### CODE REVIEW

File: Setting_up_a_study.RMD [hunk 1]
```diff
@@ -0,0 +1,53 @@
+```
```
Comment (lwjohnst86) [inline Setting_up_a_study.RMD:1]: This should be `-` minus symbol, not backtick \`

```suggestion
---
```

File: Setting_up_a_study.RMD [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+title: "Administrative and legal demands in research"
+```
```
Comment (lwjohnst86) [inline Setting_up_a_study.RMD:3]:
```suggestion
---
```

File: Setting_up_a_study.RMD [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+# Clinical studies
+      * Item 1.1 Preparing for a study
```
Comment (lwjohnst86) [inline Setting_up_a_study.RMD:6]: Nice first start! For markdown lists, you need to use either `*` or `-`, and don't need to put "Item 1.1" etc. Also, there should be an empty line before the list and should begin at the start of the line. The suggested change is below:

```suggestion

* Preparing for a study
```

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]
Great start!! I've made some suggested changes. Read them below.

File: Setting_up_a_study.RMD (hunk 3 above)
Comment (sineknorr) [reply Setting_up_a_study.RMD:6]: > Nice first start! For markdown lists, you need to use either `*` or `-`, and don't need to put "Item 1.1" etc. Also, there should be an empty line before the list and should begin at the start of the line. The suggested change is below:

Ahhhh - makes sense - feel a bit stupid for not understanding what they ment with item.....:-)

**Comment by lwjohnst86** [comment]
Ah, this, right! Thanks @sineknorr I'll look over it later today.

**Review by lwjohnst86 — APPROVED** [review]
This is a great start! I was thinking as I was looking it over whether we want to write this as a more "narrative" style, with the perspective of someone just starting a new project. Like "first you need to do this, best to set it up like that, etc." And then afterwards provide a checklist at the end? Just something to think about.

**Comment by lwjohnst86** [comment]
@sineknorr for now, I'm going to accept what you've added and join it with the main files. Your own "sinekorr" branch will still be there, so you can keep working there on this.

**Pull request merged**